idea.info.yml

```diff
@@ -100,6 +100,7 @@ dependencies:
   - openideal_faq:openideal_faq
   - openideal_idea:openideal_idea
   - openideal_user:openideal_user
+  - openideal_flavours:openideal_flavours
```

Comment:
This module shouldn't be enabled by default. It'll be enabled only in some special cases.